src/main/java/de/craften/ui/swingmaterial/MaterialMenuItem

```diff
@@ -0,0 +1,59 @@
+package de.craften.ui.swingmaterial;
+
+import de.craften.ui.swingmaterial.MaterialShadow;
+import de.craften.ui.swingmaterial.fonts.Roboto;
+import java.awt.Color;
+import java.awt.Dimension;
+import javax.swing.Action;
+import javax.swing.BorderFactory;
+import javax.swing.Icon;
+import javax.swing.JMenuItem;
+import javax.swing.UIManager;
+import javax.swing.plaf.basic.BasicMenuItemUI;
+
+/**
+ *
+ * @author Rodrigo Butzke
+ */
+public class MaterialMenuItem extends JMenuItem {
+
+    public MaterialMenuItem() {
+        init();
+    }
+    public MaterialMenuItem(Icon icon) {
+        super(icon);
+        init();
+    }
+    public MaterialMenuItem(String text) {
+        super(text);
+        init();
+    }
+    public MaterialMenuItem(Action a) {
+        super(a);
+        init();
+    }
+    public MaterialMenuItem(String text, Icon icon) {
+        super(text, icon);
+        init();
+    }
+    public MaterialMenuItem(String text, int mnemonic) {
+        super(text, mnemonic);
+        init();
+    }
+    private void init() {
+        setFont(Roboto.REGULAR.deriveFont(13f));
+        setBorder(BorderFactory.createEmptyBorder(0, MaterialShadow.OFFSET_LEFT, 0, 0));
+        setPreferredSize(new Dimension(getPreferredSize().width + MaterialShadow.OFFSET_RIGHT, 48));
+        UIManager.put("MenuItem.margin", 0);
+        UIManager.put("MenuItem.selectionBackground", new Color(238, 238, 238));
+        BasicMenuItemUI menuUI = new BasicMenuItemUI();
+        setBackground(Color.WHITE);
+        setUI(menuUI);
+    }
+    private void setHeight() {
```

Comment:
This _private_ method is only used at one place and doesn't really do what its name suggests. I think it should be dropped.